Makefile

```diff
@@ -109,16 +109,33 @@ publish-service-invoke: ## Invoke the publish service
 	&& curl -XPOST --fail $$PUBLISH_ENDPOINT
 	@$(PRINT_OK)
 
-decrypt-env:
-	source bin/decrypt-dev-env.sh
-	@$(PRINT_OK)
-
-get-secrets:
-	git submodule update --init --remote
+init-secrets:
```

Comment:
could you add comments alongside this, and make sure these commands are displayed when you run `make` and `make help`